website/sidebars.js

```diff
@@ -13,7 +13,7 @@ const docsSidebar = {
         {
             type: "category",
             label: "Installation",
-            collapsed: false,
+            collapsed: true,
```

Comment:
I liked this one uncollapsed, it avoids searching where the installation and config page are